odoo/http.py

```diff
@@ -905,7 +905,6 @@ def __init__(self, method, routing):
         self.method = method
         self.original = getattr(method, 'original_func', method)
         self.routing = routing
```

Comment:
Another idea: maybe the constructor should call
```suggestion
        self.routing = frozendict(routing)
```
instead of expecting the caller to do it